platform/platform-configuration-migration/src/test/java/org/codice/ddf/configuration/admin/ConfigurationAdminMigrationTest.java

```diff
@@ -61,7 +59,7 @@
 import org.powermock.modules.junit4.rule.PowerMockRule;
 
 @RunWith(MockitoJUnitRunner.class)
-@PrepareForTest({ConfigurationAdminMigration.class, FileUtils.class})
+@PrepareForTest({ConfigurationAdminMigration.class, FileUtils.class, Files.class})
 public class ConfigurationAdminMigrationTest {
 
     private static final Path CONFIG_FILE_PATH = Paths.get("/root/etc/exported/etc/pid.config");
```

Comment:
To be consistent, we should either make this non-static, or make all the other `Path` objects static.

May also be useful to define `private static final String CONFIG_FILENAME = CONFIG_PID + ".config";` and use that when we define the `Path` variables that need the file name.